src/libraries/System.IO.Ports/pkg/runtime.maccatalyst-x64.runtime.native.System.IO.Ports.proj

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="runtime.native.System.IO.Ports.props" />
+  <PropertyGroup>
+    <!-- This is a native package, where package baseline validation doesn't really make much sense -->
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
```

Comment:
Remove this as it's already set in the props file: https://github.com/dotnet/runtime/blob/a4ce7bca6362abc6db48a35ad88e739cb4a57bea/src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props#L14-L15

Reply:
updated. It existed in the osx file I cloned from ;(